packages/client/src/views/PrintCertificate/usePrintableCertificate.ts

```diff
@@ -123,29 +116,36 @@ export const usePrintableCertificate = (declarationId: string) => {
   const state = useSelector((store: IStoreState) => store)
   const [svg, setSvg] = useState<string>()
   const isPrintInAdvance = isCertificateForPrintInAdvance(declaration)
-  const intl = useIntl()
   const dispatch = useDispatch()
-  const languages = useSelector((store: IStoreState) =>
-    getCountryTranslations(store.i18n.languages, countries)
-  )
   const scope = useSelector(getScope)
   const canUserEditRecord =
     declaration?.event !== Event.Marriage &&
     (hasRegisterScope(scope) || hasRegistrationClerkScope(scope))
 
   useEffect(() => {
-    if (declaration)
-      getPDFTemplateWithSVG(offlineData, declaration, state).then((svg) => {
-        const svgWithFonts = addFontsToSvg(
-          svg.svgCode,
-          offlineData.templates.fonts ?? {}
-        )
-        setSvg(svgWithFonts)
-      })
+    const certificateTemplate =
```

Comment:
Not anything you need to action on but to me this whole hook feels like it's doing way too many thing starting from the fact that it fetches declarations by id and doing printing and certifying as well. Would be great to split this up in the near future